examples/kustomization/sts-example/sts-app/sts-client.yaml

```
@@ -92,3 +96,10 @@ spec:
                   items:
                     - key: ca.crt
                       path: ca.crt
+        - name: sa-token
+          projected:
+            sources:
+              - serviceAccountToken:
+                  audience: "sts.min.io"
+                  expirationSeconds: 86400
```

86400 is 1 day, and is a little long. I think AWS uses it because they are out-of-cluster.

The Kubernetes default 1 hour (720 seconds). I would stay with the default or something closer to it rather than a long period like 1 day.

```suggestion
                  expirationSeconds: 720
```

---

already set 1 hour, thanks for the input @bburky!